tests/integration/test_pulls.py

```diff
@@ -139,6 +147,31 @@ def test_repository(self):
             assert p.repository == ('sigmavirus24', 'github3.py')
 
 
+class TestPullReview(IntegrationHelper):
+    """Integration tests for the PullFile object."""
+    def get_pull_request_review(self, owner, repo, pull_number, review):
+        p = self.gh.pull_request(owner, repo, pull_number)
+
+        for pull_review in p.reviews():
+            if pull_review.id == review:
+                break
+        else:
+            assert False, "Could not find '{0}'".format(review)
+
+        return pull_review
+
+    def test_user(self):
+        """Show that a user can retrieve the user of a PR review."""
+        cassette_name = self.cassette_name('pull_reviews')
```

Comment:
So the cassette name is based entirely on the class name, so this is attempting to use a cassette called `PullReview_pull_reviews.json` and you've only added `PullRequest_pull_reviews.json`. I don't think you need to have this as a separate test. If you want to assert the `.user` attribute is a `github3.users.User` you can tuck that into the other test that you wrote.